Structure_Engine/Query/IsNull.cs

```diff
@@ -0,0 +1,469 @@
+        [Input("msg", "Optional message to be returned in addition to the generated error message.")]
+        [Output("isNull", "True if the Constraint3DOF is null.")]
+        public static bool IsNull(this Constraint3DOF constraint, string msg = "", [CallerMemberName] string methodName = "Method")
+        {
+            if (constraint == null)
+            {
+                ErrorMessage(methodName, "Constraint3DOF", msg);
+                return true;
+            }
+
+            return false;
+        }
+
+        [Description("Checks if a Constraint4DOF is null and outputs relevant error message.")]
+        [Input("constraint", "The Constraint4DOF to test for null.")]
+        [Input("methodName", "The name of the method to reference in the error message.")]
+        [Input("msg", "Optional message to be returned in addition to the generated error message.")]
+        [Output("isNull", "True if the Constraint4DOF is null.")]
+        public static bool IsNull(this Constraint4DOF constraint, string msg = "", [CallerMemberName] string methodName = "Method")
+        {
+            if (constraint == null)
+            {
+                ErrorMessage(methodName, "Constraint4DOF", msg);
+                return true;
+            }
+
+            return false;
+        }
+
+        [Description("Checks if a Case is null and outputs relevant error message.")]
+        [Input("loadCase", "The Case to test for null.")]
+        [Input("methodName", "The name of the method to reference in the error message.")]
+        [Input("msg", "Optional message to be returned in addition to the generated error message.")]
+        [Output("isNull", "True if the Case is null.")]
+        public static bool IsNull(this ICase loadCase, [CallerMemberName] string methodName = "Method", string msg = "")
+        {
+            if (loadCase == null)
+            {
+                ErrorMessage(methodName, "Case", msg);
+                return true;
+            }
+
+            return false;
+        }
+
+        /***************************************************/
+        /**** Public Methods - Interface                ****/
+        /***************************************************/
+
+        [Description("Checks if an AreaElement is null and outputs relevant error message.")]
+        [Input("element", "The AreaElement to test for null.")]
+        [Input("methodName", "The name of the method to reference in the error message.")]
+        [Input("msg", "Optional message to be returned in addition to the generated error message.")]
+        [Output("pass", "True if the AreaElement is null.")]
+        public static bool IIsNull(this IAreaElement element, string msg = "", [CallerMemberName] string methodName = "Method")
+        {
+            if (element == null)
+            {
+                ErrorMessage(methodName, "AreaElement", msg);
+                return true;
+            }
+
+            return IsNull(element as dynamic, msg, methodName);
```

Comment:
THis will never reach the method for the FEMesh after my request to move the method name last. WIll need to do type checking and call the appropriate method here unfortunately.